packages/at_client/test/monitor_test.dart

```diff
@@ -676,5 +683,32 @@ void main() {
           () async => await monitor.getQueueResponse(maxWaitTimeInMillis: 10),
           throwsA(predicate((dynamic e) => e is AtClientException)));
     });
+    group('A group of tests for monitor when useAtChops is true', () {
```

Comment:
We should also have an additional auth test for RemoteSecondary for when useAtChops is true

Reply:
authenticate and authenticateCram methods in RemoteSecondary are no longer used.
executeVerb and executeCommand methods in RemoteSecondary, delegates the call to atLookup which does the auth.
Can I explore writing tests for AtLookup auth?